charlib/cli/utils.py

```
@@ -14,22 +14,46 @@ def find_yaml_files(path) -> list:
+        if not any(value.lower().endswith(ext) for ext in ['.yml', '.yaml']):
+            return value
+        possible_yamls = find_yaml_files(value)
+        if len(possible_yamls) != 1:
+            raise ValueError(f'Unable to resolve {value} to a unique existing file')
+        with open(possible_yamls[0]) as file:
+            return yaml.safe_load(file)
+    return value
+
+
+
 def find_config(config_path, quiet=True):
```

Ruff (0.15.21)

[warning] Boolean default positional argument in function definition

(FBT002)